deepmd/model/ener.py

```diff
@@ -95,7 +101,7 @@ def _compute_input_stat (self, all_stat, protection = 1e-2) :
         self.descrpt.compute_input_stats(all_stat['coord'],
                                          all_stat['box'],
                                          all_stat['type'],
-                                         all_stat['natoms_vec'],
+                                         all_stat['natoms_vec'], # this should be the global one(dim 14) or local one(dim 11)?
```

Comment:
I'm thinking about how to organize data more reasonably, because now we are using global type map everywhere, and each descriptor deal with its own atom type